src/hooks/wallets/wallets.ts

```diff
@@ -97,3 +99,24 @@ export const isSmartContractWallet = async (wallet: ConnectedWallet) => {
 export const shouldUseEthSignMethod = (wallet: ConnectedWallet): boolean => {
   return isHardwareWallet(wallet) || isSafeMobileWallet(wallet)
 }
+
+type SigningMethods = Parameters<Safe['signTransaction']>[1]
+
+export const getSupportedSigningMethods = (
+  safeVersion: SafeInfo['version'],
+  wallet: ConnectedWallet,
+): SigningMethods[] => {
+  const EIP712_METHODS: SigningMethods[] = ['eth_signTypedData_v4', 'eth_signTypedData_v3', 'eth_signTypedData']
```

Comment:
Richard said we shouldn't use `eth_signTypedData_v3` because it was a beta version. And `eth_signTypedData_v4` is just a synonym for `eth_signTypedData`, apparently.

So in the end only `eth_signTypedData` and `eth_sign` should remain.

Reply:
Will this cause any cross-version compatability issues @germartinez? I see that [v adjustment was added for v1.0.0 support](https://github.com/safe-global/safe-core-sdk/blob/main/packages/safe-core-sdk/src/Safe.ts#L465-L470).